lib/host/wasi/inode-linux.cpp

```diff
@@ -911,34 +914,67 @@ WasiExpect<INode> INode::sockOpen(__wasi_address_family_t AddressFamily,
   }
 }
 
-WasiExpect<void> INode::sockBind(__wasi_address_family_t AddressFamily,
-                                 Span<const uint8_t> Address,
-                                 uint16_t Port) noexcept {
-  Variant<sockaddr, sockaddr_in, sockaddr_in6> ServerAddr;
-  size_t Size;
+struct SockEmptyAddr {};
+using VarAddrT = std::variant<SockEmptyAddr, sockaddr, sockaddr_in,
+                              sockaddr_in6, sockaddr_un>;
```

Comment:
I found we have use `Variant<sockaddr_storage, sockaddr_in, sockaddr_in6, sockaddr_un> SockAddr;` in the following code. What's the difference between this and the above one? Can we unify them into only one data structure?